packages/vue/examples/__tests__/transition/Transition.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { E2E_TIMEOUT, setupPuppeteer } from '../e2eUtils'
+import path from 'path'
+
+function nextFrame(cb: () => void) {
+  requestAnimationFrame(() => {
+    requestAnimationFrame(cb)
+  })
+}
+
+describe('e2e: Transition', () => {
+  const { page, html, click, classList } = setupPuppeteer()
+  const baseUrl = `file://${path.resolve(
+    __dirname,
+    '../../transition/index.html'
+  )}`
+  const duration = 50,
+    buffer = 10
+  const container = '#test'
+
+  test(
+    'basic transition',
+    async () => {
+      await page().goto(baseUrl)
+      await page().waitFor('#app')
+      expect(await html(container)).toBe('<div class="test"></div>')
+
+      await click('button')
+      expect(await classList('#test div')).toStrictEqual([
```

Comment:
I need some help.
It's here.I run test with this.The first state of dom sometime error or sometime correct.I think maybe it is caused by the unknown reason of puppeteer.But I dont find some help message with test transition in puppeteer.

Reply:
This is some message with error.
```
 "test",
    "v-leave-active",
-   "v-leave-from",
+   "v-leave-to",
```